src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerSubmissionControllerTests.cs

```diff
@@ -30,6 +31,7 @@
 
     public class ProducerSubmissionControllerTests : SimpleUnitTestBase
```

Comment:
@will-hope98 sorry if I missed it, but I can see tests for Post_ValidModel_ShouldRedirectToTaskList but not for Post_ValidModel_ShouldRedirectToCheckAnswers in the case that RedirectToCheckAnswers is true

Reply:
Added